src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import os from "node:os";
+
+const __confDir = os.homedir() + "/.baseplate";
```

Comment:
Does the `/` work here on windows? `path.sep` sometimes is required, or `path.join()`

Reply:
it works in Windows, but will use path.sep to be on safer side